SwanCustomEnvironments/swancustomenvironments/scripts/builders/venv.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash
 
-# Create the environment
-python3 -m venv ${ENV_PATH} 2>&1
+# Create a middle layer for installing Spark extensions, putting them apart from the user environment
+uv venv $SWAN_ENV --seed 2>&1
+source $SWAN_ENV/bin/activate
+uv pip install "ipykernel==${IPYKERNEL_VERSION}"
+SWAN_PACKAGES_PATH=$(python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
+deactivate
+
+# Create the environment using uv, if possible
```

Comment:
Delete this comment, it does not bring any extra information. Also, it is not there in the acc-py equivalent code block.